README.md

```diff
@@ -13,9 +13,12 @@ Other notetaking apps like Obsidian and Vscode Foam have this feature and after
 ## Usage
 `:Daily` opens todays daily note
 
-`:NextNote` opens the next daily note (won't go past todays date)
+`:NextDaily` opens the next daily note
+  - Won't create a new file for the next day if one does not exist
```

Comment:
Actually it should, as long as you're on a daily notes file: https://asciinema.org/a/ieacov45A3nOmSSiY5xApwedH
same for the below change